docs/components/modeler/desktop-modeler/troubleshooting.md

```diff
@@ -171,6 +171,20 @@ DEBUG=* ZEEBE_NODE_LOG_LEVEL=DEBUG GRPC_VERBOSITY=DEBUG GRPC_TRACE=all camunda-m
 </TabItem>
 </Tabs>
 
+## Desktop Modeler does not start on Ubuntu 24 / modern Linux
+
+Modern Linux operating systems introduce restrictions on user namespaces, a sandboxing (isolation) mechanism the modeler uses.
+
+The effect that you see is an error message like the following at the start of the application:
+
+```sh
+$ ./camunda-modeler
+[46193:1114/170934.837319:FATAL:setuid_sandbox_host.cc(163)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that [...]/camunda-modeler-[...]-linux-x64/chrome-sandbox is owned by root and has mode 4755.
+zsh: trace trap (core dumped)  ./camunda-modeler
+```
+
+To remedy this you have to configure your system to allow the sandboxing (again) or disable the sandbox altogether (dangerous, not recommended). [This resource](https://github.com/camunda/camunda-modeler/issues/4695#issuecomment-2478581677) explains the available options in depth.
```

Comment:
I wouldn't phrase it like that, this might bring confusion to those who started from v24
```suggestion
To remedy this you have to configure your system to allow the sandboxing or disable the sandbox altogether (dangerous, not recommended). [This resource](https://github.com/camunda/camunda-modeler/issues/4695#issuecomment-2478581677) explains the available options in depth.
```

Reply:
Also is it a good practice to link to a discussion under an issue instead of place the recommended solution directly here and add the link to that discussion as an additional source of information?

Reply:
> Also is it a good practice to link to a discussion under an issue instead of place the recommended solution directly here and add the link to that discussion as an additional source of information?

Consider yourself as a user: Would you want the recommended solution here, and link to alternatives (outside of this topic)? Especially as a "I'm not root and I don't care" solution is available that we _do not recommend_, but may be the only option for some users I'm actually torn. I think we should show different resolutions, and allows users to decide. Hence I rather linked to the external source.

Reply:
I tried to inline the recommended option, but it feels odd (as it hides crucial information). What I could imagine instead is to link directly to the respective comment.

Reply:
Please have a look.

Reply:
In my opinion from a user perspective, it is always good to have the documentation give you the recommended solution straight away instead of redirect to other sources and make you find the correct answer yourself. Of course there are many solution to this problem as you listed them under that linked issue, but if there is any "best practice" or "recommended solution" from our side, we should give it directly to the user and as an extension (for those who still have issues) there is the link to alternative options for which most of users won't have to bother.
Looking at your both solutions, they both look good to me, cause they are both clearly indicate the recommendation. If the recommended solution should be placed in the docs directly or not this is a subjective matter - to me personally I like it to be in the doc instead of linked, even if it looks a little bulky and takes space.

Reply:
> to me personally I like it to be in the doc instead of linked, even if it looks a little bulky and takes space.

Let's consider this as a follow-up and merge the shorter solution for now.